tgui/packages/tgui-panel/styles/components/Chat.scss

```
@@ -84,10 +84,16 @@ $color-bg-section: base.$color-bg-section !default;
 
 .Chat__highlight {
   color: #000;
+  // Спаны с background-clip: text делают текст прозрачным через
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)